buildbot/osuosl/master/config/builders.py

```diff
@@ -3169,9 +3172,23 @@
                     "-DCMAKE_C_COMPILER_LAUNCHER=ccache",
                     "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache"],
                 extra_stage2_cmake_args=[
-                    "-DLLVM_ENABLE_LLD=True",
-                    "-DCMAKE_C_FLAGS='-march=rva20u64'",
-                    "-DCMAKE_CXX_FLAGS='-march=rva20u64'"]
+                    util.Interpolate(f"-DLLVM_NATIVE_TOOL_DIR=%(prop:builddir)s/stage1.install/bin"),
+                    "-DLLVM_BUILD_TESTS=True",
+                    "-DLLVM_EXTERNAL_LIT=/home/buildbot-worker/lit-on-qemu-system-rva20.py",
```

Comment:
I've put the script in llvm-zorg and added an option to checkout llvm-zorg. This is similar to what the sanitizer builders to, grabbing scripts from this repo. I had initially hoped to use the buildbot file transfer ability, but ultimately any way I looked at to try to get repo contents relative to the loaded zorg buildbot configuration seemed potentially brittle (cwd for the buildmaster isn't necessarily within an llvm-zorg checkout - e.g. like with the local testing config it might have a symlink to the zorg dir within an llvm-zorg checkout).